src/main/java/se/sundsvall/myrepresentative/api/MandatesResource.java

```diff
@@ -0,0 +1,94 @@
+		})
+	@PostMapping(value = "/mandates", consumes = APPLICATION_JSON_VALUE, produces = ALL_VALUE)
+	ResponseEntity<Void> createMandate(
+		@Parameter(name = "municipalityId", description = "Municipality id", example = "2281") @ValidMunicipalityId @PathVariable final String municipalityId,
+		@Valid @RequestBody final CreateMandate request) {
+		// TODO implement
+		return ResponseEntity.status(HttpStatus.NOT_IMPLEMENTED).build();
+	}
+
+	@Operation(summary = "Get mandates",
+		responses = {
+			@ApiResponse(responseCode = "200",
+				description = "Successful Operation",
+				useReturnTypeSchema = true)
+		})
+	@GetMapping(value = "/mandates", produces = APPLICATION_JSON_VALUE)
+	ResponseEntity<List<MandateDetails>> getMandates(
+		@Parameter(name = "municipalityId", description = "Municipality id", example = "2281") @ValidMunicipalityId @PathVariable final String municipalityId,
+		@Parameter(name = "grantorPartyId", description = "PartyId of the grantor (person or organization)", example = "2facc7a8-69e1-4988-9b3d-4da6cefab701") @RequestParam(required = false) @ValidUuid(nullable = true) final String grantorPartyId,
+		@Parameter(name = "granteePartyId", description = "PartyId of the grantee of the mandate", example = "2facc7a8-69e1-4988-9b3d-4da6cefab702") @RequestParam(required = false) @ValidUuid(nullable = true) final String granteePartyId,
+		@Parameter(name = "signatoryPartyId", description = "PartyId of the signatory", example = "2facc7a8-69e1-4988-9b3d-4da6cefab703") @RequestParam(required = false) @ValidUuid(nullable = true) final String signatoryPartyId) {
+		// TODO implement
+		return ResponseEntity.status(HttpStatus.NOT_IMPLEMENTED).build();
+	}
+
+	@Operation(summary = "Update mandate",
+		responses = {
+			@ApiResponse(responseCode = "200",
+				description = "Ok",
+				useReturnTypeSchema = true)
+		})
+	@PatchMapping(value = "/mandates", consumes = APPLICATION_JSON_VALUE, produces = ALL_VALUE)
+	ResponseEntity<MandateDetails> updateMandate(
+		@Parameter(name = "municipalityId", description = "Municipality id", example = "2281") @ValidMunicipalityId @PathVariable final String municipalityId,
+		@Valid @RequestBody final UpdateMandate updateRequest) {
+		// TODO implement
+		return ResponseEntity.status(HttpStatus.NOT_IMPLEMENTED).build();
+	}
+
+	@Operation(summary = "Delete mandate, performs a soft delete",
+		responses = {
+			@ApiResponse(responseCode = "204",
+				description = "No Content",
+				useReturnTypeSchema = true)
+		})
+	@DeleteMapping(value = "/mandates/{id}", produces = ALL_VALUE)
+	ResponseEntity<Void> deleteMandate(
+		@Parameter(name = "municipalityId", description = "Municipality id", example = "2281") @ValidMunicipalityId @PathVariable final String municipalityId,
+		@Parameter(name = "id", description = "Id of the mandate", example = "2facc7a8-69e1-4988-9b3d-4da6cefab704") @ValidUuid @PathVariable final String id) {
+		// TODO implement
+		return ResponseEntity.status(HttpStatus.NOT_IMPLEMENTED).build();
```

Comment:
Change to `return ResponseEntity.status(HttpStatus.NOT_IMPLEMENTED).header(CONTENT_TYPE, ALL_VALUE). build()` as this resource produces ALL_VALUE